client/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -190,6 +200,12 @@ export class JetpackConnectNotices extends Component {
 				);
 				return noticeValues;
 
+			case INSTALL_FAILURE:
+				noticeValues.text = translate(
+					'We were unable to install Jetpack. Please upgrade your WordPress version.'
```

Comment:
This is not a case we have actually seen, is it?

Reply:
No :) It is guarding against possible cases of the same installations issue. Maybe worth keeping, at least until the testing is over?

Reply:
No, it is guarding against possible cases of the same error on success issue. Maybe worth keeping, at least until the testing is over? That said, it does not need to be part of UI... :)